zebra-chain/Cargo.toml

```diff
@@ -14,14 +14,16 @@ chrono = "0.4"
 futures = "0.3"
 hex = "0.4"
 lazy_static = "1.4.0"
+rand_core = "0.5.1"
 ripemd160 = "0.8.0"
-secp256k1 = "0.17.2"
+secp256k1 = { version = "0.17.2", features = ["serde"] }
 serde = { version = "1", features = ["serde_derive"] }
-sha2 = "0.8"
+sha2 = {git="https://github.com/RustCrypto/hashes", rev="30b416a", features=["compress"]}
```

Comment:
:+1: can we have a comment above this with a reference to some upstream tracking issue?

Reply:
There's no upstream tracking issue, we're just waiting for them to push a release to crates.io or tag a commit.